lib/modules/showImages.js

```diff
@@ -1268,7 +1266,48 @@ function generateIframe(options) {
 	element.independent = true;
 
 	keepMediaVisible(iframeNode);
-	makeMediaIndependentOnResize(element, iframeNode);
+	makeMediaIndependentOnResize(element, element);
+
+	// Handle resize for iframe
+	let initialWidth, initialDiagonal, minWidth, left, top, ratio;
+
+	function getDiagonal(x, y) {
```

Comment:
Since documents don't have a fixed aspect ratio, keeping this is unnecessary. IMO it's more user-friendly to resize the iframe to basically w = offsetLeft, h = offsetTop (though this would mean only allow resize initiated from the right bottom corner).